common/src/main/java/org/figuramc/figura/animation/Animation.java

```diff
@@ -140,6 +141,25 @@ public void playCode(float minTime, float maxTime) {
     // -- lua methods -- // 
 
 
+    @LuaWhitelist
+    @LuaMethodDoc("animation.getChannels")
+    public LuaTable getChannels() {
```

Comment:
I am guessing this is because it is called "channel" in BlockBench?

Reply:
its because thats what figura refers to them as (AnimationChannel)